linode/manager
# refactor: [M3-6264] - MUI v5 Migration - SRC > Features > Events

## Description 📝
MUI v5 CSS migration for SRC > Features > Events

## Major Changes 🔄
- Eliminated default exports
- Eliminated makeStyles
- Renamed props
- Created .styles files

## Preview 📷
| Before  | After   |
| --- | --- |
| ![Screenshot 2023-08-22 at 3 05 02 PM](https://github.com/linode/manager/assets/139489745/4f8aae65-39fd-48d1-a873-c19d655f663c) | ![Screenshot 2023-08-22 at 3 03 44 PM](https://github.com/linode/manager/assets/139489745/23a3e45e-c333-44d2-9e27-f95de5dd3423) |

## How to test 🧪
1. Navigate to http://localhost:3000/events
2. Ensure the styling is consistent
